main.nf

```diff
@@ -45,19 +52,20 @@ vim: syntax=groovy
  */
 
 // Pipeline version
-version = 0.1
+version = 1.0
 
 // Reference genome index
 params.genome = 'GRCh37'
 params.gtf   = params.genomes[ params.genome ].gtf
 params.bed12 = params.genomes[ params.genome ].bed12
-params.bowtie2 = params.genomes[ params.genome ].bowtie2
-params.bowtie_miRBase_mature = params.genomes.bowtie_miRBase_mature
-params.bowtie_miRBase_hairpin = params.genomes.bowtie_miRBase_hairpin
+params.index = params.genomes[ params.genome ].bowtie2
+params.mature  = params.miRBase_mature[ params.genome ].bowtie
+params.hairpin = params.miRBase_hairpin[ params.genome ].bowtie
 params.name = "miRNA-Seq Best practice"
+params.outdir = './results'
 
 // Input files
-params.input = "data/*.fastq.gz"
+params.reads = "data/*{1,2}*.fastq.gz"
```

Comment:
We can keep the current version I guess? Since the scripts are treating the reads as SE anyway?  It can be easily switched to PE mode if needed.